### ISSUE TITLE
Start-IpPbxDockerDeployment.ps1 erkennt Ende der Installation nicht.

### ISSUE THREAD

**Issue by ghost** [Swyx/swyxware-docker#1]
Hallo,
bei einem Installationsversuch mit Hilfe der Start-IpPbxDockerDeployment.ps1 blieb diese bei der Meldung "SwyxWare is being configured. This may take a while..." stehen. Nach einiger Zeit konnte ich aber sehen, daß die IpPbx*-Prozesse liefen. Ich kann mich auch mit der SwyxWare Administrations-MMC an dem Server anmelden. 

`docker container logs --tail 25 ippbx01` liefert folgendes, die Installation scheint erfolgreich zu sein:
> Verb Task completed.
> Verb TaskQueue progress 100
> .Info Finished running tasks.
> 
> Trace file: C:\ProgramData\Swyx\Traces\CDSClient.IpPbxConfig-20170928-145118.log
> VERBOSE: Creating 'AlreadyConfigured.dat'...
> Configuration completed
> VERBOSE: Keeping container alive...
> 
> TimeGenerated          EntryType Source          Message
> -------------          --------- ------          -------
> 9/28/2017 2:55:14 PM Information IpPbxConfig     Unattended configuration wi...
> 9/28/2017 2:55:14 PM Information IpPbxConnectSrv SwyxRemoteConnector service...
> 9/28/2017 2:55:10 PM Information IpPbx3pcc       SwyxCTI+ service started su...
> 9/28/2017 2:55:07 PM Information IpPbxFaxSrv     SwyxFax Server started succ...
> 9/28/2017 2:55:06 PM Information IpPbxFaxPrtGate SwyxFax Printer Gateway sta...
> 9/28/2017 2:54:57 PM Information LinkMgr         LinkMgr started successfully.
> 9/28/2017 2:54:56 PM Information PhoneMgr        PhoneMgr started successfully.
> 9/28/2017 2:54:55 PM Information ConferenceMgr   ConferenceMgr registered at...
> 9/28/2017 2:54:54 PM     Warning IpPbxSrv        SwyxServer has at least one...
> 9/28/2017 2:54:54 PM Information IpPbxCDS        Server runs in Enterprise m...
> 9/28/2017 2:54:53 PM Information IpPbxCDS        Server runs in Enterprise m...
> 9/28/2017 3:55:14 PM     Warning IpPbxSrv        SwyxServer has at least one...
> 9/28/2017 4:55:14 PM     Warning IpPbxSrv        SwyxServer has at least one...
> 9/28/2017 5:55:14 PM     Warning IpPbxSrv        SwyxServer has at least one...

Möglicherweise liegt das Problem an der Art, wie nach der Erfolgsmeldung rund um Zeile 446
`$ContainerLogLastLines = docker container logs --tail 5 $IpPbxContainerName`
gesucht wird. Ich vermute, daß die Meldung _Configuration completed_, bei mir in Zeile 7 der Ausgabe, nie gefunden wurde.

**Comment by martinh2011** [comment]
Ja, das wird es sein. Der Mechanismus ist nicht wirklich schön. Kurzfristig habe ich mal die Anzahl der Zeilen, die für den Check durchsucht werden, erhöht.
